src/structures/Manager.ts

```diff
@@ -229,10 +222,11 @@ export class Manager extends EventEmitter {
      * @returns {Promise<Player>}
      */
     public async createPlayer(options: PlayerOptions): Promise<Player> {
-        let player = this.players.get(options.guild);
+        var player = this.players.get(options.guild);
```

Comment:
Is there any specific reason for changing let to var?

Reply:
They both work same the only difference is that when using `var` player will be undefined from the start of the function body.

Reply:
since the function is creating player isn't it better to keep it let then

Reply:
Would it matter?